src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-list/rbd-trash-list.component.ts

```diff
@@ -201,13 +201,14 @@ export class RbdTrashListComponent implements OnInit {
     const namespace = this.selection.first().namespace;
     const imageId = this.selection.first().id;
     const expiresAt = this.selection.first().deferment_end_time;
+    const isExpired = moment().isAfter(expiresAt);
```

Comment:
This variable is used by the dialog template at https://github.com/ceph/ceph/pull/38154/files#diff-2b6c1f8d6be785a4afc4072dcfa30921e7eadcac7d93cc1daa8c42324d766389R45.

> Does this work? I mean, is this updated when the expiry date passes?

If you plan to keep the dialog open for minutes/days/weeks until the expiration date then i have to answer no, but IMO this will never happen.

Reply:
No problem then! Thank you for the explanation.